enginetest/queries/queries.go

```diff
@@ -10971,97 +10971,36 @@ var IndexPrefixQueries = []ScriptTest{
 					{0, "a", "a"},
 				},
 			},
-			{
```

Comment:
How confident are you that we have suitable coverage for these plans elsewhere?

Reply:
all of these are preserved, moved into `IndexQueryPlans`